pkg/services/publicdashboards/service/service.go

```diff
@@ -342,6 +342,32 @@ func (pd *PublicDashboardServiceImpl) Delete(ctx context.Context, orgId int64, u
 	return nil
 }
 
+func (pd *PublicDashboardServiceImpl) HandleDashboardDeleted(ctx context.Context, dashboard *dashboards.Dashboard) error {
```

Comment:
I would call it `DeleteByDashboardDeletion`

Reply:
so, based on the methods names we currently have I think it makes more sense to put something like `DeleteByDashboardDeletion`. But it's up to you

Reply:
Yeah I see it follows our convention, but it sounds really weird lol. @evictorero can you be the tie-breaker here?

Reply:
If we want to follow the convention it should be something like `ActionByParameters`, this case would be `DeleteByDashboard`. This way you can reuse and call it from different places, not only Dashboard Deletion.